src/api/app/models/user.rb

```diff
@@ -42,6 +42,7 @@ class User < ApplicationRecord
 
   belongs_to :owner, class_name: 'User'
   has_many :subaccounts, class_name: 'User', foreign_key: 'owner_id'
+  has_many :bs_requests
```

Comment:
I would leave it as it is, to be honest. I can also imagine what it means by not having a `created_` at the beginning.